snap/info.go

```diff
@@ -1422,9 +1423,48 @@ func (app *AppInfo) SecurityTag() string {
 	return AppSecurityTag(app.Snap.InstanceName(), app.Name)
 }
 
+func snapAppNameFromDesktopFile(desktopFile string) (string, error) {
```

Comment:
shouldn't we use code from desktopentry for this?

Reply:
Moved parsing to desktopentry, Thanks!